Gruntfile.js

```diff
@@ -213,6 +213,22 @@ module.exports = function(grunt) {
 					dest: 'test/preload/fixtures/library-same-dest'
 				},
 				libraries: '**'
+			},
+
+			'library_custom_uglify_params': {
+				options: {
+					resources: 'test/preload/fixtures/library-custom-uglify-params',
+					dest: 'tmp/preload/library_custom_uglify_params',
+					compress: {
+						uglifyjs: {
+							mangle: false,
+							output: {
+								ascii_only: true
```

Comment:
In this configuration the `comments` option is not changed, so it should use the [default pattern](https://github.com/SAP/grunt-openui5/blob/8ef82a510aed7159c81433d1896fbe7073535426/tasks/preload.js#L34).